src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -132,4 +141,107 @@ public CommandBase turnDegrees(double degrees, double speed) {
         cmd.setName("Turn Degrees");
         return cmd;
     }
+
+    public CommandBase slowTurn(boolean direction) {
+        CommandBase cmd = new RunCommand(() -> {
+            if (direction == true) {
+                driveTrain.arcadeDrive(0, 0.4);
+            } else if (direction == false) {
+                driveTrain.arcadeDrive(0, -0.4);
+
+            }
+        }, this);
+        cmd.setName("turing");
+        return cmd;
+    }
+
+    public CommandBase visionAlignDegrees() {
+        CommandBase cmd = new CommandBase() {
```

Comment:
Make sure to use an initializer block to add the subsystem as a dependency - one of the flaws with command base itself.